(SHIP-1481) Increase default LimitDefault for CCIP

HelloKashif opened the pull request:
LimitDefault of 8M is the most common value used in chains so setting this as the default

RensR commented on core/chains/evm/config/toml/defaults/fallback.toml:
```diff
@@ -32,7 +32,7 @@ Mode = 'BlockHistory'
 PriceMax = '115792089237316195423570985008687907853269984665.640564039457584007913129639935 tether'
 PriceMin = '1 gwei'
-LimitDefault = 500_000
+LimitDefault = 8_000_000
```
Let's use this nice underscore_format_in other places as well :)

RensR replied:
I mean for the 8m values you put everywhere, not unrelated entires in the fallback. Please undo any unrelated changes

HelloKashif committed:
Increase LimitDefault and LimitMax

HelloKashif committed:
Increase LimitDefault and LimitMax

amaechiokolobi committed:
fix limit default tests (#932)

## Motivation

Fixed the test failures 

## Solution

Updated the relevant values in the test files and docs

HelloKashif committed:
Update changeset

amaechiokolobi committed:
Increase LimitDefault and LimitMax in CONFIG.md (#948)

## Motivation

Failing tests due to mismatch in CONFIG.md

## Solution

Update LimitDefault and LimitMax in CONFIG.md

---------

Co-authored-by: Kashif Siddiqui <kashif.siddiqui@smartcontract.com>

amaechiokolobi committed:
Increase LimitDefault and LimitMax (#952)

## Motivation

Failing tests due to mismatch in CONFIG.md

## Solution

Update LimitDefault and LimitMax in CONFIG.md

---------

Co-authored-by: Kashif Siddiqui <kashif.siddiqui@smartcontract.com>##
Motivation


## Solution

amaechiokolobi committed:
updated config docs (#953)

## Motivation


## Solution

amaechiokolobi committed:
Merge branch 'kashif/limit-default' of github.com:smartcontractkit/ccip into kashif/limit-default

amaechiokolobi committed:
Merge branch 'ccip-develop' into kashif/limit-default

amaechiokolobi committed:
updated config docs

amaechiokolobi committed:
Merge branch 'kashif/limit-default' of github.com:smartcontractkit/ccip into kashif/limit-default

amaechiokolobi committed:
updated config docs

amaechiokolobi committed:
added changeset

amaechiokolobi committed:
added changeset

amaechiokolobi committed:
reverted changes to help.txtar

jmank88 commented on testdata/scripts/node/db/help.txtar:
IIRC these need to be exact matches

jmank88 commented on core/config/docs/chains-evm.toml:
```diff
@@ -160,9 +160,9 @@ PriceMax = '115792089237316195423570985008687907853269984665.6405640394575840079
 # LimitDefault sets default gas limit for outgoing transactions. This should not need to be changed in most cases.
 # Some job types, such as Keeper jobs, might set their own gas limit unrelated to this value.
-LimitDefault = 500_000 # Default
+LimitDefault = 8000000 # Default
```
Let's retain the format for readability:
```suggestion
LimitDefault = 8_000_000 # Default
```

amaechiokolobi committed:
removed whitespace

amaechiokolobi committed:
removed whitespace

amaechiokolobi committed:
Merge branch 'ccip-develop' into kashif/limit-default

amaechiokolobi committed:
Merge branch 'ccip-develop' into kashif/limit-default

amaechiokolobi committed:
Merge branch 'kashif/limit-default' of github.com:smartcontractkit/ccip into kashif/limit-default

amaechiokolobi committed:
increased node funding

amaechiokolobi committed:
increased node funding

amaechiokolobi committed:
enabled DEUBUG_RESTY

RensR commented:
Please remove the changesets as we no longer use them. Also please us the `8_000_000` format over `8000000`

amaechiokolobi committed:
removed changeset

skudasov committed:
Merge branch 'ccip-develop' into kashif/limit-default

amaechiokolobi committed:
updated docs

amaechiokolobi committed:
Merge branch 'kashif/limit-default' of github.com:smartcontractkit/ccip into kashif/limit-default

amaechiokolobi committed:
updated txtar

amaechiokolobi committed:
added empty newline

amaechiokolobi committed:
added empty newline

amaechiokolobi committed:
increased node funding

amaechiokolobi committed:
increased node funding

amaechiokolobi committed:
updated expected gas to 8m

amaechiokolobi committed:
increased node funding

amaechiokolobi committed:
increased node funding for all tests

amaechiokolobi committed:
increased operator contract funding

amaechiokolobi committed:
Merge branch 'ccip-develop' into kashif/limit-default

jmank88 commented on .changeset/README.md:
Why are all of these changeset files being deleted?

simsonraj replied:
+1

amaechiokolobi replied:
@RensR mentioned earlier to remove them as they are no longer being used https://github.com/smartcontractkit/ccip/pull/664#issuecomment-2178238991

RensR replied:
I meant remove the ones you created, not every single one that exists

amaechiokolobi replied:
my mistake I misunderstood the message 🤦 I'll add these back now

amaechiokolobi replied:
the changesets were re-added when merging, so this issue has been corrected @RensR

jmank88 commented on core/chains/evm/config/config_test.go:
```diff
@@ -232,8 +232,8 @@ func TestChainScopedConfig_GasEstimator(t *testing.T) {
 	assert.Equal(t, assets.GWei(1), ge.PriceMin())
-	assert.Equal(t, uint64(500000), ge.LimitDefault())
-	assert.Equal(t, uint64(500000), ge.LimitMax())
+	assert.Equal(t, uint64(8000000), ge.LimitDefault())
+	assert.Equal(t, uint64(8000000), ge.LimitMax())
```
Go supports the `_` format too:
```suggestion
	assert.Equal(t, uint64(8_000_000), ge.LimitDefault())
	assert.Equal(t, uint64(8_000_000), ge.LimitMax())
```

simsonraj commented on .github/workflows/integration-tests.yml:
```diff
@@ -535,6 +535,7 @@ jobs:
         env:
           BASE64_CCIP_CONFIG_OVERRIDE: ${{ steps.set_override_config.outputs.base_64_override }}
           TEST_BASE64_CCIP_CONFIG_OVERRIDE: ${{ steps.set_override_config.outputs.base_64_override }}
+          DEBUG_RESTY: true
```
do we need to let this on?

simsonraj commented on core/chains/evm/config/toml/defaults/fallback.toml:
```diff
@@ -5,9 +5,9 @@ FinalityDepth = 50
+LogKeepBlocksDepth = 100_000
 # CCIP uses paging when removing logs to avoid pushing too much pressure on the database
-LogPrunePageSize = 10000
+LogPrunePageSize = 10_000
```
I would keep this PR clean only for the changes related to the problem & not touch these fields for now

simsonraj commented on core/chains/evm/config/toml/defaults/fallback.toml:
```diff
@@ -79,4 +79,4 @@ DeltaCJitterOverride = '1h'
 
 [OCR2.Automation]
-GasLimit = 5400000
+GasLimit = 5_400_000
```
Same here, please dont change the formats for other values as part of this PR

simsonraj commented on core/internal/features/features_test.go:
```diff
@@ -385,7 +385,7 @@ func TestIntegration_DirectRequest(t *testing.T) {
 			n, err := b.NonceAt(testutils.Context(t), operatorContracts.user.From, nil)
 			require.NoError(t, err)
-			tx = cltest.NewLegacyTransaction(n, sendingKeys[0].Address, assets.Ether(100).ToInt(), 21000, big.NewInt(1000000000), nil)
+			tx = cltest.NewLegacyTransaction(n, sendingKeys[0].Address, assets.Ether(801).ToInt(), 21000, big.NewInt(1000000000), nil)
```
odd value 801?

amaechiokolobi replied:
The test required 800 eth + ~1 gwei, I will change this to 1000 so it is cleaner

simsonraj commented on integration-tests/testconfig/default.toml:
```diff
@@ -12,7 +12,7 @@ image = "public.ecr.aws/chainlink/chainlink"
 
 [Common]
-chainlink_node_funding = 0.5
+chainlink_node_funding = 1
```
Is this enough to cover the tests?

amaechiokolobi replied:
It seems to be fine for the tests, but I will increase it to 8 to align with the gas limit changes. I had this set as I previously had issues with the smoke tests before but now that its fixed it should be fine

amaechiokolobi replied:
Set this to 2, as I was getting many errors when it was set to 4

amaechiokolobi committed:
fixed number formatting

amaechiokolobi committed:
increased chainlink node funding

amaechiokolobi committed:
increased test node account funding to 1000

amaechiokolobi committed:
disabled debug_resty

amaechiokolobi committed:
Merge branch 'kashif/limit-default' of github.com:smartcontractkit/ccip into kashif/limit-default

amaechiokolobi committed:
changed number formatting

amaechiokolobi committed:
fixed number formatting

amaechiokolobi committed:
fixed number formatting

amaechiokolobi committed:
fixed number formatting

amaechiokolobi committed:
decreased chainlink node funding

simsonraj committed:
Merge branch 'ccip-develop' into kashif/limit-default

amaechiokolobi committed:
reverted unrelated changes

amaechiokolobi committed:
Merge branch 'ccip-develop' into kashif/limit-default

amaechiokolobi committed:
Merge branch 'ccip-develop' into kashif/limit-default

amaechiokolobi committed:
Merge branch 'ccip-develop' into kashif/limit-default

mateusz-sekara commented on .github/workflows/integration-tests.yml:
```diff
@@ -584,6 +584,7 @@ jobs:
         env:
           BASE64_CCIP_CONFIG_OVERRIDE: ${{ steps.set_override_config.outputs.base_64_override }}
           TEST_BASE64_CCIP_CONFIG_OVERRIDE: ${{ steps.set_override_config.outputs.base_64_override }}
+          DEBUG_RESTY: false
```
Why do we need this change?

amaechiokolobi replied:
@mateusz-sekara We only had it enabled to debug a prior issue. Or do you think I should remove the line entirely?

mateusz-sekara reviewed (approved):
Looks good, one minor question about change in the CI files

jasonmci reviewed (approved):
Approved.

amaechiokolobi committed:
Merge branch 'ccip-develop' into kashif/limit-default
